CRM/Utils/Mail/Incoming.php

```diff
@@ -285,49 +285,16 @@ public static function formatAddress($address) {
     return $name . "<{$address->email}>";
   }
 
-  /**
```

Comment:
This is not used anywhere in universe.

Reply:
If you want to pull that into its own PR, while it's not straightforward a quick grep looks true and could get that merged lickety-split.

Reply:
Sure, PR #27011